modules/renter/download.go

```diff
@@ -121,16 +121,15 @@ func (d *download) getPiece(chunkIndex, pieceIndex uint64) []byte {
 // run performs the actual download. It spawns one worker per host, and
 // instructs them to sequentially download chunks. It then writes the
 // recovered chunks to w.
-func (d *download) run(w io.Writer) error {
-	var received uint64
-	for i := uint64(0); received < d.fileSize; i++ {
+func (d *download) run(chunkIndex uint64, received uint64, w io.Writer) (uint64, uint64, error) {
```

Comment:
sorry, I meant the returned values. They should be named, or the docstring should at least be updated